homeassistant/components/vera/config_flow.py

```diff
@@ -10,8 +10,13 @@
 from homeassistant import config_entries
 from homeassistant.const import CONF_EXCLUDE, CONF_LIGHTS, CONF_SOURCE
 from homeassistant.core import callback
+from homeassistant.helpers.entity_registry import EntityRegistry
 
-from .const import CONF_CONTROLLER, DOMAIN
+from .const import (  # pylint: disable=unused-import
```

Comment:
Why not fix the pylint error?

Reply:
It's a bug in pylint. We use the domain constant but pylint doesn't read that.